utils/cliutils/commandsflags.go

```diff
@@ -1829,7 +1829,7 @@ var commandFlags = map[string][]string{
 	},
 	Docker: {
 		buildName, buildNumber, module, Project,
-		serverId, skipLogin, threads, detailedSummary, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
+		serverId, skipLogin, threads, detailedSummary, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly, vuln,
```

Comment:
What's the reason for this flag addition?

Reply:
@eyalbe4 we noticed that this flag wasn't passed to JFrog CLI Security, so I added it here.

Reply:
:+1: